classes/class-wc-connect-help-view.php

```diff
@@ -310,6 +322,50 @@ public function page() {
 			);
 		}
 
+		/**
+		 * @return array
+		 */
+		protected function get_tax_health_item() {
+			if ( class_exists( 'WC_Taxjar' ) ) {
+				return [
+					'state'              => 'warning',
```

Comment:
How about using "error" instead in this an all of the other cases? Automated taxes can't work so I'd say that's an error in the context of "can we use them" (even if it's not severe because e.g. the TaxJar extension is active and handling taxes).

Reply:
I see what you're saying. I was thinking that the `error` state seemed so "catastrophic".
If a message is displayed in this section, it is likely that the automated tax never worked for the merchant. So I didn't want to be so harsh with the error state.
What are your thoughts on that?

Reply:
Yeah, I was also trying to figure out what the best error type would be... It seems `error` is more prevalent in other usages in `classes/class-wc-connect-help-view.php`, and `warning` is only displayed where either we can't yet verify something or it's a staging site.

Based on this, going with `error` seems more consistent to me but I have no strong feelings on this. :)